terraform/projects/app-ecs-services/task-definitions/prometheus-server.json

```diff
@@ -3,7 +3,7 @@
     "name": "prometheus",
     "image": "prom/prometheus",
     "cpu": 2048,
-    "memory": 8192,
+    "memoryReservation": 8192,
```

Comment:
I've opened #9 to deal with this - we can leave it for now but it'll get changed again